datapusher/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM keitaro/base:0.4
```

Comment:
Is there no existing datapusher image? If we are forced to build our own, then we should build our own image from its own repository. We shouldn't be committing a datapusher Dockerfile here.

Reply:
It seems you use a local datapusher in the [inventory app](https://github.com/GSA/inventory-app/blob/inventory_ckan_2.8/config/production.ini#L158) for development. Also in [datagov-deploy](https://github.com/GSA/datagov-deploy/blob/develop/ansible/roles/software/ckan/inventory/templates/etc/ckan/production.ini#L201)
I din't find a docker image in [GSA's Docker hub](https://hub.docker.com/u/datagov).
This repo seems the best place to test the upgrade.
Does GSA have a repository for the datapusher image?

Reply:
Not sure but it seems [in datagov-deploy you just run datapusher locally](https://github.com/GSA/datagov-deploy/blob/develop/ansible/roles/software/ckan/inventory/defaults/main.yml#L11-L24) in the inventory machine.

Reply:
If you prefer @adborden we can start a new repo for GSA datapusher to work independently.